src/plugins/data_source_management/public/components/data_source_option.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiBadge } from '@elastic/eui';
+import { SelectedDataSourceOption } from './data_source_multi_selectable/data_source_filter_group';
+
+export interface ShowDataSourceOptionProps {
+  item: SelectedDataSourceOption;
+  defaultDataSource: string | null;
+}
+
+export const ShowDataSourceOption: React.FC<ShowDataSourceOptionProps> = ({
```

Comment:
Can we simply name it DataSourceOption and then DataSourceOptionProps

Reply:
Hi We  have a type called DataSourceOption already. Let me rename it to DataSourceOptionItem as it shows each item